assets/csidriveroperators/aws-ebs/hypershift/mgmt/lease_role.patch.yaml

```diff
@@ -0,0 +1,15 @@
+- op: "add"
+  path: "/rules/-"
+  value:
+    apiGroups:
+    - coordination.k8s.io
+    resources:
+    - leases
```

Comment:
This looks like a bug in base/05_clusterrole.yaml - no operator / operand should have `ClusterRole` to read/edit Leases in **all** namespaces, it should be in `Role`. And if we move it to 03_role.yaml, then we may not need to patch it here.

Reply:
Well until we remove it from https://github.com/openshift/aws-ebs-csi-driver-operator/blob/master/assets/rbac/snapshotter_role.yaml, removal of this clusterrole causes operator to be degraded. So we will have to do a follow up.